src/bookshelf/src/bksw_io_names.erl

```diff
@@ -61,7 +61,7 @@ sha_to_hex_str(<<SHA:160/big-unsigned-integer>>) ->
 -spec write_path(string() | binary(), string() | binary()) -> binary().
 write_path(Bucket, Path) ->
     Root = bksw_conf:disk_store(),
-    {T1, T2, T3} = erlang:now(),
+    {T1, T2, T3} = erlang:timestamp(),
```

Comment:
Because `erlang:timestamp()` doesn't provide the same uniqueness guarantees as erlang:now(), I think we might actually want to add some randomness or use `erlang:unique_integer()` here otherwise we may get collisions.  Same for your use in `generate_id` in bksw_req.sql